docs/technical_reference/costing/watertap_costing.rst

```diff
@@ -0,0 +1,168 @@
+            return cost_my_unit_model
+
+    m = pyo.ConcreteModel()
+    m.fs = idc.FlowsheetBlock(dynamic=False)
+    m.fs.costing = WaterTAPCosting()
+
+    m.fs.my_unit_1 = MyUnitModel()
+    
+    # The `default_costing_method_attribute` on the
+    # unit model is checked, and the function
+    # `cost_my_unit_model` returned then build the costing block.
+    # This method also adds the `my_unit` global parameter block,
+    # so the global costing parameter m.fs.costing.my_unit.fixed_capital_cost
+    # is the same for all instances of MyUnitModel.
+    m.fs.my_unit_1.costing = idc.UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+    )
+
+    m.fs.my_unit_2 = MyUnitModel()
+
+    # Here everythin as before, but the global parameter block
+    # m.fs.costing.my_unit is not re-built. 
+    m.fs.my_unit_2.costing = idc.UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+    )
+
+
+Costing WaterTAP Models
+-----------------------
+
+The WaterTAPCostingData class includes variables and constraints necessary to calculate process-wide costs:
+
+=============================================  ====================  ===============================================  ==============================================================================
+                 Cost                               Variable                 Name                                         Description
+=============================================  ====================  ===============================================  ==============================================================================
+Total capital cost                              :math:`C_{ca,tot}`    ``total_capital_cost``                          Total capital cost
+Unit capital cost                               :math:`C_{ca,u}`      ``aggregate_capital_cost``                      Unit processes capital cost
+Total operating cost                            :math:`C_{op,tot}`    ``total_operating_cost``                        Total operating cost for unit process
+Maintenance, labor, and chemical cost           :math:`C_{mlc}`       ``maintenance_labor_chemical_operating_cost``   Maintenance cost for unit process
+Total annualized cost                           :math:`C_{annual}`    ``total_annualized_costs``                      Total cost on a annualized basis
+=============================================  ====================  ===============================================  ==============================================================================
+
+Calculations for each of these costs are presented below.
+
+Costing Index and Technoeconomic Factors
+----------------------------------------
+
+Default costing indices are provide with the WaterTAP Costing Package, but the user is free to modify these for their needs.
+
+WaterTAP uses the Consumer Price Index (CPI) to help account for the time-value of investments and are used in the capital
```

Comment:
Hmm.. I thought we used the Chemical Engineering Plant Cost Index (CEPCI)

Reply:
You are correct. I copied this sentence from the ZO costing documentation and fixed it in both places.